libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -31,14 +31,22 @@ object Aggregates : TemplateGroupBase() {
         include(Maps, CharSequences, ArraysOfUnsigned)
     } builder {
         inline()
-        specialFor(ArraysOfUnsigned) { inlineOnly() }
+        specialFor(ArraysOfUnsigned) {
+            inlineOnly()
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(Maps) {
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(CharSequences) {
```

Comment:
This should be enough:
```
sample("samples.collections.Collections.Aggregates.all")
specialFor(CharSequences) {
    sample("samples.text.Strings.all")
}
```
should be enough.

The DSL means that _"samples.collections.Collections.Aggregates.all" is the sample for all receivers, but for CharSequences there is a different sample_

So basically the second `sample(...)` overrides the previous.